src/data/calendar.js

```diff
@@ -0,0 +1,36 @@
+const mongoCollections = require('../config/mongoCollections');
+const boards = mongoCollections.boards;
+const error_handler = require('../errors/error-handler'); 
+
+/*! ics file function library written by Travis Krause and Kyle Hornberg */
```

Comment:
Add a reference link to where you found this. Is this a node library that could just be installed instead of having the longest line of code I've ever seen? We don't use var, we use let.

Reply:
This is the link https://github.com/nwcell/ics.js I didn't see a way to install the library, and this was how they used it in their code so I assumed this was how it should be used, I can check it out though. Also, I swapped var with let for everything I could, but the saveAs var doesnt work as a let so I didnt know what to do.

Reply:
This feels a lil iffy, if we just copy this whole ass library like this.

Try this npm package: https://www.npmjs.com/package/ics

Reply:
I rewrote the function to use the npm package